.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           cp -rf ${{ github.workspace }}/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
       - name: Install dependencies
-        run: python -m pip install cibuildwheel~=2.16.0 toml
+        run: python -m pip install cibuildwheel~=2.16.0 tomli
```

Comment:
If this is in the pyproject.toml (and setup.py as a runtime dep) there will be no need to set it here and everywhere.

Reply:
This is not in the pyproject.toml in favour of using tomllib builtin package for py3.11 and py3.12. But it'd be much cleaner to add tomli to pyproject that also preserves the toml parser consistency for all python versions.

Reply:
A toml parser is required for configuring `pyproject.toml`. This configuration occurs before calling `setup.py` via `PL_BACKEND="lightning_sim" python scripts/configure_pyproject_toml.py`.

I had to replace `tomli` (w/o `tomli-w`) with `tomlkit` as the `configure_pyproject_toml.py` reads and writes into `pyproject.toml`, and `tomli` is a read-only toml parser.